apps/acrs/behaviours/save-image.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+const _ = require('lodash');
+const config = require('../../../config');
+const Model = require('../models/image-upload');
+
+const fileSizeNum = size => size.match(/\d+/g)[0];
+
+module.exports = name => superclass => class extends superclass {
+  process(req) {
+    if (req.files && req.files[name]) {
+      // set image name on values for filename extension validation
+      // N:B validation controller gets values from
+      // req.form.values and not on req.files
+      req.form.values[name] = req.files[name].name;
+      req.log('info', `Submission ID: ${req.sessionModel.get('submissionID')},
+                       Processing image: ${req.form.values[name]}`);
+    }
+    super.process.apply(this, arguments);
+  }
+
+  locals(req, res, next) {
+    const maxNum = fileSizeNum(config.upload.maxFileSize);
+    const maxSize = config.upload.maxFileSize.match(/[a-zA-Z]+/g)[0].toUpperCase();
+
+    return Object.assign({}, super.locals(req, res, next), {
+      maxFileSize: `${maxNum} ${maxSize}`
+    });
+  }
```

Comment:
Tested with these changes, but it allows the uploading of files larger than 25Mb

Reply:
Applied the changes, but set the `maxFileSizeInBytes` value to exactly 25000000 in `config.js` using the standard value instead of the binary value. When using the binary value, it allows the uploading of files up to 26.2 MB.